app/views/isa_assays/_assay_samples.html.erb

```diff
@@ -1,8 +1,4 @@
-<%
-=begin%>
- The dynamic table where users can interact with
-<%
-=end%>
+<%= # The dynamic table where users can interact with %>
```

Comment:
the `#` should directly follow the `<%` with no `=` or space
e.g. `<%# The dynamic table where users can interact with %>`